editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -220,8 +220,8 @@ xdescribe('conditionals in the navigator', () => {
           regular-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2
             conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/then-then-div-then
               regular-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/then-then-div
-            conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/else-case-else
-              synthetic-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/else-case-attribute
+            conditional-clause-utopia-storyboard-uid/scene-aaa/containing-div/conditional1/conditional2/false-case-else
```

Comment:
Hmmm why has this ended up as `false-case-else`, whereas the tests in `ui-jsx-canvas-conditionals.spec.browser2.tsx` have been changed to `false-case-false-case`?

Reply:
I just missed it and the test is disabled, so it did not break the build.

Reply:
I reverted my changes and added a separate task to fix and enable this test